docs/src/main/sphinx/release/release-465.md

```diff
@@ -0,0 +1,160 @@
+# Release 465 (20 Nov 2024)
+
+## General
+
+* Add the {func}`cosine_similarity` function for dense vectors. ({issue}`23964`)
+* Add support for reading geometries in [EWKB
+  format](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry)
+  with the {func}`ST_GeomFromBinary` function. ({issue}`23824`)
+* Add support for parameter of `bigint` type for the {func}`repeat` function. ({issue}`22867`)
+* Add support for the `ORDER BY` clause in a windowed aggregate function. ({issue}`23929`)
+* {{breaking}} Change the data type for `client_info` in the MySQL event
+  listener to `MEDIUMTEXT`. ({issue}`22362`)
+* Improve performance of queries with selective joins. ({issue}`22824`)
```

Comment:
I just spent like 10 minutes looking for this, can release notes mention names of the properties when we change their default values, like `enable-dynamic-filtering` in this case? Users might want to remove them from their configs if they don't change the default value anymore.